platform/marvell-armhf/sonic-platform-nokia/debian/sonic-platform-nokia-7215.install

```diff
@@ -1,5 +1,6 @@
 nokia-7215_plt_setup.sh usr/sbin
 7215/scripts/nokia-7215init.sh  usr/local/bin
 7215/service/nokia-7215init.service  etc/systemd/system
+7215/service/fstrim.timer/override.conf  /etc/systemd/system/fstrim.timer.d
```

Comment:
Ah, if installing in a package, it's better to have it be in `/lib/systemd/system/fstrim.timer.d`. Also, I'd recommend naming it something besides `override.conf`, in case there happens to be some conflict (which is unlikely).

Reply:
For /etc/systemd, I followed your example as well as that of the file, nokia-7215init.service.   I'm not sure when to use /etc/systemd vs /lib/systemd and what the convention really is.  Are you sure we need to change this?

Reply:
The /etc/systemd directory is meant for service files and overrides that are created by the system administrator. These are generally not installed through a package. The /lib/systemd directory are for service files that do come from a package. See [this](https://manpages.debian.org/bullseye/systemd/systemd.unit.5.en.html#UNIT_FILE_LOAD_PATH) for a more detailed description.

I would recommend following the convention, at least for consistency. At the very least, since override.conf is a common name for service file overrides, and since this is coming from a package, I'd highly recommend renaming this file.

Reply:
@saiarcot895, Please check if the latest update is satisfactory.  Thanks